### ISSUE TITLE
Genmon Health(GPIO 10) and GenServer Status

### ISSUE THREAD

**Issue by sambeetm** [jgyates/genmon#577]
Use the template below if you have an issue or want to report a bug. If you have a question or a feature request you can ignore the questions below. This is a template so feel free to remove any of this text. Try to include the info below when applicable.

***NOTE:*** If you are having issues with your serial connection, please read this page before posting:

https://github.com/jgyates/genmon/wiki/3.6---Serial-Troubleshooting

If you are having other issues, please see the following page:

https://github.com/jgyates/genmon/wiki/3.5---General-Troubleshooting

You can search past issues by removing the "is:open" keywords from the search window and adding any keywords that are relevant to your issue.

If you need to send you logs and registers to the developer, if your email is setup and working properly you can click send your logs on the About page in the web interface.

### Expected Behavior
I am using GPIO Output addon to display various statuses using the LEDs. Just found that the GenServer is not responding. Not able to login to the web interface locally. Yet, the LED connected to GPIO 10 (which is Genmon Health) is not lit. Is not GenServer health is part of Genmon health. I would expect the GPIO 10 should stay High when Gen Server is down or unresponsive.

### Actual Behavior

GenServer is down yet GPIO 10 is low.

### Steps to Reproduce (including precondition)

{Please write here}

### Screenshot or Pictures relating to the problem (if possible)

{Please write here}

### Your Environment

- Generator Model: {Please write here}
- Genmon Logs: Submit via the About page
- Genmon Version: {Please write here}

**Comment by jgyates** [comment]
genmon.py monitors a few things to determine the health status: Active serial or serial over TCP communications, all required threads within gnemon.py are running as expected, the modbus initialization sequence is complete, and the genmon.log file is not increasing in size. The health metric is reported on the monitor page of the web interface. This health metric does not cover any other programs like genserv.py or any add on program. In my setup I use a network monitor (https://www.nagios.org/) to monitor the web server is active, in addition to monitoring other items on my network. The file /OtherApps/check_monitor_system.py is a nagios plugin to monitor genmon.py. Other built in nagios plug ins can check if a web server is active.

In short, this health check is not inclusive of everything. It is limited by design.
